dapr/dapr-with-istio/README.md

```diff
@@ -0,0 +1,132 @@
+# Dapr with Istio
+
+This example is the result of a POC showing how to have [Dapr](https://dapr.io/) besides a service-mesh setup, like [Istio](https://istio.io/latest/)
+
+## Goal
+- Create a sample application, using dapr for inter-cluster communication
+- Use darp mTLS for dapr related communication (clients to control plane, client to client)
+- Expose the application using Istio mechanisms
+- Use Istio mTLS for istio related communication (user to gateway, gateway to service)
+
+## Requirements
+- A base Kyma cluster with service-mesh installed.
+- [helm v3](https://helm.sh/)
+- kubectl
+
+## System setup
+
+### Install Dapr with helm
+Use the official Helm charts to install Dapr. However, we create the namespace beforehand and disable istio sidecar injection using the label `istio-injection: disabled`.
+
+```bash
+helm repo add dapr https://daprio.azurecr.io/helm/v1/repo
+
+kubectl apply -f ./manifests/dapr-system.yaml
+helm install dapr dapr/dapr --namespace dapr-system
+```
+
+### Setup Darp StateStore (Redis)
+```bash
+helm repo add bitnami https://charts.bitnami.com/bitnami
+helm repo update
+
```

Comment:
Maybe for the example let's not mix dapr-system and its control plane with any persistence and  provision it in separate ns ?